docs/ui/foundations/icons.tsx

```diff
@@ -0,0 +1,198 @@
+import { iconSize, palette, theme } from '@expo/styleguide';
+import React from 'react';
+import { Clipboard } from 'react-feather';
+
+export type IconProps = {
+  size?: number | string;
+  style?: object;
+  fill?: string;
+  stroke?: string;
+  color?: string;
+  className?: string;
+  width?: number | string;
+  height?: number | string;
+};
+
+export {
+  ChevronDownIcon,
+  DiscordIcon,
+  ErrorIcon,
+  InfoIcon,
+  WarningIcon,
+  GithubIcon,
+  TwitterIcon,
+} from '@expo/styleguide';
+
+// TODO(cedric): check if we can move these to styleguide too
+
+export const ClipboardIcon = (props: IconProps) => {
+  const { size = iconSize.regular, color = theme.icon.default } = props;
+  return <Clipboard size={size} color={color} />;
+};
+
+export const SDKIcon = (props: IconProps) => {
+  const { size = iconSize.regular, stroke = theme.icon.default, fill = palette.dark.white } = props;
```

Comment:
```suggestion
  const { size = iconSize.regular, stroke = theme.icon.default, fill = theme.palette.white } = props;
```

We now export all colors on `theme.palette`. This will switch between `palette.light.white` and `palette.dark.white` in light/dark modes. In this particular case, it wont change the color, since `white` is `#FFF` for both. But! just a little more consistent than having to choose a light/dark side.